### DISCUSSION TITLE
Is there any way to install openssl 1.1?

### DISCUSSION THREAD

**Discussion by Darth-Bobo** [amazonlinux/amazon-linux-2022#157] [Q&A] [answered]
I've been checking out the previous AL2022 releases for use with RabbitMQ and now that OpenSSL 3 is baked into the AMI I can no longer install RabbitMQ because even the latest version of Erlang requires OpenSSL 1.1.

I raised this with the devs and they basically told me to go away as 1.1 is used all over the place so they won't put the effort in to test against 3 to support an Amazon release.

I don't care which version we use as our cluster is only used internally, so if we want to move forward our only option at the moment is to migrate off Amazon Linux unless I can install OpenSSL 1.1.
 
As it was in the preview AMIs, I assume it is tucked away in a repo somewhere - but where?

**Comment by stewartsmith** [comment, marked as answer]
You'll need a RabbitMQ built for AL2022 (although a package request for us is also okay), as AL2022 is only shipping with OpenSSL 3.

We do not have plans to ship OpenSSL 1.1 as part of AL2022. It was available in some earlier preview versions to assist us in moving the rest of the OS to OpenSSL 3.

**Comment by softloft38p-michael** [comment]
We also need OpenSSL 1.1 and/or a RabbitMQ package for Amazon Linux 2022: how can we get working RabbitMQ built?
